src/components/Page/README.md

```diff
@@ -391,7 +391,7 @@ Use for layouts that benefit from more screen width, such as wide tables or list
 <Page
   fullWidth
   title="Orders"
-  primaryAction={{content: 'Create order'}}
+  primaryAction={{content: 'Create order', icon: PlusMinor}}
```

Comment:
Again, this icon won't be displayed in the current design language. It only show up when using the new design language and on small screen (turns into an icon-only button)